components/Footer.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <div class="absolute inset-x-0 bottom-0 text-sm text-center">
+    Developed with <span>❤</span>️ by <a href="http://sergiocastro.me" class="text-gray-900 underline">Sérgio Castro</a> using Nuxt.
```

Comment:
Could you add a new line featuring a link to the contributors (https://github.com/sergiocastrovale/tonality/graphs/contributors) and another one to the repo itself (https://github.com/sergiocastrovale/tonality)?